src/components/Authentication/Authentication.js

```diff
@@ -1,10 +1,56 @@
+import { useEffect, useState } from 'react';
 import { useAuth0 } from '@auth0/auth0-react';
 
 import HeaderButton from '../HeaderButton/HeaderButton';
 
+const AUTH0_USER_DOMAIN = process.env.REACT_APP_AUTH0_DOMAIN;
+
 const Authentication = ({ style }) => {
-  const { error, isLoading, isAuthenticated, user, loginWithRedirect, logout } = useAuth0();
+  const {
+    error,
+    isLoading,
+    isAuthenticated,
+    user,
+    getAccessTokenSilently,
+    loginWithRedirect,
+    logout,
+  } = useAuth0();
+
+  const [userMetaData, setUserMetaData] = useState(null);
+
+  useEffect(() => {
+      const getUserMetaData = async () => {
+        const domain = AUTH0_USER_DOMAIN;
+        try {
+          const accessToken = await getAccessTokenSilently({
+            authorizationParams: {
+               audience: `https://${domain}/api/v2/`,
+               scope: 'read:current_user',
+            },
+           });
```

Comment:
You need to get this token and put it to the Axios config.
You can do it in this way `axiosClient.interceptors.request.use((config) => {
        config.headers.Authorization = `Bearer ${token}``